README.md

```diff
@@ -3,9 +3,11 @@ KestrelHttpServer
 
 [![Join the chat at https://gitter.im/aspnet/KestrelHttpServer](https://badges.gitter.im/aspnet/KestrelHttpServer.svg)](https://gitter.im/aspnet/KestrelHttpServer?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-AppVeyor: [![AppVeyor](https://ci.appveyor.com/api/projects/status/nr0s92ykm57q0bjv/branch/dev?svg=true)](https://ci.appveyor.com/project/aspnetci/KestrelHttpServer/branch/dev)
+[![Travis build status][travis-badge]](https://travis-ci.org/aspnet/KestrelHttpServer/branches)
```

Comment:
Why the change from https://travis-ci.org/aspnet/KestrelHttpServer to https://travis-ci.org/aspnet/KestrelHttpServer/branches?